### ISSUE TITLE
Docker images are not published

### ISSUE THREAD

**Issue by necrolyte2** [nuagenetworks/nuage-kubernetes#116]
[This](https://github.com/nuagenetworks/nuage-kubernetes/blob/master/kubernetes-standalone-deployment/kubernetes-1-installation.rst) document contains the following instructions but there are no instructions on where to get these tar files(I have found various Dockerfile's and was able to build some of them but the `nuage/vrs` docker image is nowhere to be found)
```
docker load -i nuage-master-docker.tar
docker load -i nuage-cni-docker.tar
docker load -i nuage-vrs-docker.tar
docker load -i nuage-infra-docker.tar
```

Is there any reason that these images have not been published to Dockerhub or some other repo. At a very minimum a single script somewhere that would build all the required images?

**Comment by ic3cool** [comment]
29 days and not a word around this...

**Comment by pdellaert** [comment]
Hi @necrolyte2, @ic3cool,
 
The `nuage-master-docker`, `nuage-cni-docker` and `nuage-infra-docker` images are currently not available on Dockerhub, but it is something we can look at to build and release them.
 
The `nuage-vrs-docker` image however is our Nuage VRS container which is build with each release of the Nuage VSP product. This container is only obtainable through our release portal for customers.

**Comment by necrolyte2** [comment]
I think it makes sense for you to publish the containers that are public to dockerhub(or similar)

Most workflows that involve docker do not usually involve scp'ing tar files around.

I can understand that the vrs plugin is proprietary so cannot be published

**Comment by pdellaert** [comment]
@necrolyte2 yes, it does make sense to make them public, we just need to integrate the publication process with our CI/CD and properly validate that process :)
